src/app/feedback/feedback-form/feedback-form.component.ts

```diff
@@ -49,8 +50,9 @@ export class FeedbackFormComponent implements OnInit {
   }
 
   private getHeaderText(): string {
-    const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
-    return el ? el.innerHTML : '';
+    // const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
```

Comment:
<img width="1156" alt="screen shot 2017-05-07 at 2 43 19 pm" src="https://cloud.githubusercontent.com/assets/20037396/25783989/94f5be08-3333-11e7-9f03-455ae2f5b331.png">
after changing to `document.body` on master